geom.go

```diff
@@ -415,6 +416,31 @@ func NewPolygonFromFlatPoints (shell []float64) (*Geometry, error){
 	return PolygonFromGeom(ext)
 }
 
+func (geom *Geometry) PolygonToFlatPoints(out []float64) ([]float64, error) {
+	shell, shellErr := geom.Shell()
+	if shellErr != nil {
+		return nil, shellErr
+	}
+	ptr := cGEOSGeom_getCoordSeq(shell.g)
+	if ptr == nil {
+		return nil, Error()
+	}
+	//cs := coordSeqFromPtr(ptr)
```

Comment:
Commented code

Reply:
In fact the original code was better since it set the finalizer